salt/journal/config/etc-nginx-sites-enabled-journal.conf

```diff
@@ -57,4 +57,14 @@ server {
 
     access_log /var/log/nginx/journal.access.log combined_with_time;
     error_log /var/log/nginx/journal.error.log notice;
+
+    error_page 404 /error/404.html;
+    error_page 410 /error/410.html;
+    error_page 400 401 402 403 405 406 407 408 409 411 412 413 414 415 416 417 418 421 422 423 424 426 428 429 431 444 451 495 496 497 /error/4xx.html;
```

Comment:
I don't think there's a better way to do this (ie a wildcard). I took the list of codes from Wikipedia with the Nginx extras, but had to remove `499 Client Closed Request` as it doesn't allow it.